app/service_invite/rest.py

```diff
@@ -32,20 +32,14 @@
 register_errors(service_invite)
 
 
-def _create_service_invite(invited_user, invite_link_host):
+def _create_service_invite(invited_user, nonce):
 
     template_id = current_app.config["INVITATION_EMAIL_TEMPLATE_ID"]
 
     template = dao_get_template_by_id(template_id)
 
     service = Service.query.get(current_app.config["NOTIFY_SERVICE_ID"])
 
-    token = generate_token(
-        str(invited_user.email_address),
-        current_app.config["SECRET_KEY"],
-        current_app.config["DANGEROUS_SALT"],
-    )
```

Comment:
We don't need to generate the token anymore in this case?  I'm assuming that's because the `nonce` is what actually provides that for us now?

Reply:
I removed that code because we were not using the token anywhere. Once I changed how the nonce was used, it was unnecessary.